docs/_ug/placeholders/INDEX.md

```diff
@@ -4,7 +4,7 @@ The INDEX of an item is the number to the left of the item name in the Item List
 ```info
 * INDEX is a whole number larger than 0
 * INDEX is dependent on the item list currently displayed in the Item List Box
-* Do not include a thousands-separator in INDEX
+* Do not include a thousands-separator `,` in INDEX
```

Comment:
A comma is not the only thousands-separator.

```suggestion
* Do not include a thousands-separator (e.g. `,`) in INDEX
```

Reply:
Makes sense. Will change accordingly.